client/src/Movies/Movie.js

```diff
@@ -11,15 +11,16 @@ export default class Movie extends Component {
 
   componentDidMount() {
     // change this line to grab the id passed on the URL
-    const id = 1;
+    // console.log(this.props)
+    const id = Number(this.props.match.params.id);
     this.fetchMovie(id);
   }
 
   fetchMovie = id => {
     axios
       .get(`http://localhost:5000/api/movies/${id}`)
       .then(response => {
-        this.setState(() => ({ movie: response.data }));
+        this.setState(() => ({ movie: response.data }), () => console.log(this.state.movie));
```

Comment:
This is great, using console logs to ensure the state is being set in the way you want.